web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/field--os2loop-shared-subject.html.twig

```diff
@@ -0,0 +1,83 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a field.
+ *
+ * To override output, copy the "field.html.twig" from the templates directory
+ * to your theme's directory and customize it, just like customizing other
+ * Drupal templates such as page.html.twig or node.html.twig.
+ *
+ * Instead of overriding the theming for all fields, you can also just override
+ * theming for a subset of fields using
+ * @link themeable Theme hook suggestions. @endlink For example,
+ * here are some theme hook suggestions that can be used for a field_foo field
+ * on an article node type:
+ * - field--node--field-foo--article.html.twig
+ * - field--node--field-foo.html.twig
+ * - field--node--article.html.twig
+ * - field--field-foo.html.twig
+ * - field--text-with-summary.html.twig
+ * - field.html.twig
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see template_preprocess_field()
+ *
+ * @ingroup themeable
+ */
+#}
+{% set title_classes = [
+    label_display == 'visually_hidden' ? 'visually-hidden',
+    label_display == 'inline' ? 'os2loop-title-text-inline',
+    label_display != 'inline' ? 'os2loop-title-text',
+  ]
+%}
+{% if label_hidden %}
+  {% if multiple %}
+    {% for item in items %}
+      <div{{ item.attributes }}>{{ item.content }}</div>
+    {% endfor %}
+  {% else %}
+    {% for item in items %}
+      {{ item.content }}
+    {% endfor %}
+  {% endif %}
+{% elseif label_display == 'inline' %}
+  <span{{ attributes }}>
+    <span{{ title_attributes.addClass(title_classes) }}>{{ label }}:</span>
+    {% if multiple %}
+      <span>
+    {% endif %}
+    {% for item in items %}
+      <span{{ item.attributes.addClass(item.content['#options'].entity.os2loop_taxonomy_color_picker.value, 'shared-subject') }}>{{ item.content }}</span>
```

Comment:
Check that this value actually exists and consider moving it to a variable to improve readability.

Reply:
Set variable